doc/SIDEBAR_MOD.md

```diff
@@ -475,63 +487,123 @@ mapped as closely as possible to the spectrum of colors, with one exception - va
 "normal" value or range always use white (`c_white`) when the value is within normal.
 
 
-## `phrases`
+## `flags`
+
+Widgets can use flags to specify special behaviors:
+
+```json
+{
+  "id": "my_widget",
+  "type": "widget",
+  "style": "text",
+  "label": "My Widget",
+  "var": "my_widget_var",
+  "flags": [ "W_LABEL_NONE", "W_DISABLED" ]
+}
+```
+
+Here are some flags that can be included:
 
-Some widgets can take advantage of "phrases" - definitions for what text/values to display and
+| Flag id            | Description
+|---                 |---
+| `W_LABEL_NONE`     | Prevents the widget's label from being displayed in the sidebar
+| `W_DISABLED`       | Makes this widget disabled by default (only applies to top-level widgets/layouts)
+| `W_DYNAMIC_HEIGHT` | Allows certain multi-line widgets to dynamically adjust their height
+
+
+# Phrases and conditions
+
+Widgets can take advantage of "phrases" - definitions for what text/values to display and
 how to display them. These take the form of a nested object containing several optional fields:
 
 ```json
 {
   "id": "bp_status_indicator_template",
   "type": "widget",
-  "style": "phrase",
+  "style": "text",
   "phrases": [
-    { "id": "bitten", "text": "bitten", "sym": "B", "color": "yellow" },
-    { "id": "infected", "text": "infected", "sym": "I", "color": "pink" },
-    { "id": "broken", "text": "broken", "sym": "%", "color": "magenta" },
-    { "id": "splinted", "text": "splinted", "sym": "=", "color": "light_gray" },
-    { "id": "bandaged", "text": "bandaged", "sym": "+", "color": "white" },
-    { "id": "disinfected", "text": "disinfected", "sym": "$", "color": "light_green" },
-    { "id": "bleeding", "text": "bleeding", "value": 0, "sym": "b", "color": "light_red" },
-    { "id": "bleeding", "text": "bleeding", "value": 11, "sym": "b", "color": "red" },
-    { "id": "bleeding", "text": "bleeding", "value": 21, "sym": "b", "color": "red_red" }
+    { "id": "bitten", "text": "bitten", "sym": "B", "color": "yellow", "condition": { ... } },
+    { "id": "infected", "text": "infected", "sym": "I", "color": "pink", "condition": { ... } },
+    { "id": "broken", "text": "broken", "sym": "%", "color": "magenta", "condition": { ... } },
+    { "id": "splinted", "text": "splinted", "sym": "=", "color": "light_gray", "condition": { ... } },
+    { "id": "bandaged", "text": "bandaged", "sym": "+", "color": "white", "condition": { ... } },
+    { "id": "disinfected", "text": "disinfected", "sym": "$", "color": "light_green", "condition": { ... } },
+    { "id": "bleeding", "text": "bleeding", "value": 0, "sym": "b", "color": "light_red", "condition": { ... } },
+    { "id": "bleeding", "text": "bleeding", "value": 11, "sym": "b", "color": "red", "condition": { ... } },
+    { "id": "bleeding", "text": "bleeding", "value": 21, "sym": "b", "color": "red_red", "condition": { ... } }
```

Comment:
```suggestion
    { "id": "bitten", "text": "bitten", "sym": "B", "color": "yellow", "condition": "..." },
    { "id": "infected", "text": "infected", "sym": "I", "color": "pink", "condition": "..." },
    { "id": "bandaged", "text": "bandaged", "sym": "+", "color": "white", "condition": "..." }
```

A gentler introduction with a reduced example would be better here I think. And since "conditions" can be either strings or objects, I prefer the plainer form of ellipses, since they're still valid JSON (the `{ ... }` gets ugly-highlighted by the markdown viewer).

This feature is complex enough to warrant a bigger section with several examples, but that can always be saved for a future PR. Meanwhile maybe include a reference from the docs to the ones in `data/mods/TEST_DATA/widgets.json`? Those were an "a-ha" moment for me.